ccpp/data/GFS_typedefs.meta

```diff
@@ -2367,23 +2367,23 @@
   standard_name = kinematic_surface_upward_sensible_heat_flux_of_fire
   long_name = kinematic surface upward sensible heat flux of fire
   units = K m s-1
-  dimensions = (horizontal_loop_extent)
```

Comment:
@grantfirl Note to self: This was messed up in https://github.com/NOAA-EMC/fv3atm/pull/815

Reply:
This fix causes the fire behavior RT to change results. This restores the correct functionality for the two-way fire coupling.